docs/guides/migrating-from-Camunda-Platform.md

```diff
@@ -533,47 +487,32 @@ The ```bpmn message name``` is used in both products and doesn't need migration.
 
 ## Adjusting your DMN models
 
-For Camunda Cloud, [a former community extension](https://github.com/camunda-community-hub/dmn-scala), built by core Camunda developers, is productized. This engine has a higher coverage of DMN elements. This engine can execute DMN models designed for Camunda Platform 7, however, there are some small differences which can be assessed looking at [this issue](https://github.com/camunda/camunda-modeler/issues/2525).
+For Camunda 8, [a former community extension](https://github.com/camunda-community-hub/dmn-scala), built by core Camunda developers, is productized. This engine has a higher coverage of DMN elements. This engine can execute DMN models designed for Camunda 7, however, there are some small differences listed below.
```

Comment:
I think from Modeling perspective there are two different types again, right? One DMN that works on C7 only and one that works on C8 only? Should we mention how it can be manually migrated?

@MaxTru can maybe give more context

Reply:
Camunda 7 DMN files also work on Camunda 8 DMN Engine, **if** the Camunda specifics were not set: (history time to live, version tag, expression languages different than FEEL (basically what is already mentioned here). 

So IMO the text proposed here is already accurate.


Context for Modeling: from Modeling perspective we differentiate DMN for C7 and DMN for C8 diagrams by custom attributes which we set as part of the definitions:
```
<definitions ...  xmlns:modeler="http://camunda.org/schema/modeler/1.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.0.0">
```
=> the respective DMN Modeling editor does then offer / not offer to model the C7 custom attributes (e.g., history time to live)

Reply:
Looks like we can keep the text for now (Which might not be perfect anyway)?

Reply:
Hmm, as a User I would still ask:
"How would I open a C7 DMN as a C8 DMN to make it "compatible" if I have one of these attributes?" Like how do I go to the C8 mode in modeler?

Reply:
True, but (just as for BPMN) we do not have an "easy" answer in terms of "click this migration button". So the answer could only be to manually clean up the diagram (in the XML), which I believe would not be satisfying.

So if this is a need, then we should maybe prioritize to offer some semi-automated / guided migration feature soon I would say. WDYT?

cc @christian-konrad

Reply:
I can look at if the migration tooling (https://github.com/camunda-community-hub/camunda-platform-to-cloud-migration) is able to do the required changes - but I am sure this will not be there before launch (as the biggest complexity is to get the environment running with DMN files and probably use a different set of dms-js or whatever). @MaxTru Unless you have a good idea for a quick fix on https://github.com/camunda-community-hub/camunda-platform-to-cloud-migration/tree/main/camunda-modeler-plugin-platform-to-cloud-converter to be applicable to DMN files and make necessary changes?

Reply:
Moving discussion to https://github.com/camunda-cloud/camunda-cloud-documentation/issues/780 so we can close it here